cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,43 @@
+import faker from 'faker';
+import HomeAndCataloguePageObject
```

Comment:
in this task, you should use all the capabilities of the page object model. So create CartPageObject (or something like this) and use functions that you will create in this class in your flow

Reply:
Fixed